src/pages/comparison/comparison.component.tsx

```diff
@@ -1,53 +1,49 @@
 import React, { useEffect } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
-import { findPetsById } from '../../store/actions/comparison.action';
+import { asyncGetPeysById } from '../../store/actions/comparison.action';
 
-import './comparison.css';
+import './comparison.scss';
 
 import { Logo } from '../../shared/logotype/logo';
 import { Characteristics } from './components/characteristics.component';
 
-import { Pet } from './interfaces/pet.interface';
+import {  ComparisonProps, PetsToCompareList, RootState } from './interfaces/pet.interface';
 
-type ComparisonProps = {
-  name: string;
-  match: {
-    params: {
-      ids: string;
-    };
-  };
-};
-
-interface RootState {
-  data: object;
-  petsToCompare: Array<Pet>;
-}
-
-export function Comparison (props: ComparisonProps): React.ReactElement {
-  const { ids } = props.match.params;  //extract pets to compare id from pathname
+export const Comparison: React.FC<ComparisonProps> = props => {
+  const { ids } = props.match.params; //extract pets to compare id from pathname
 
   const dispatch = useDispatch();
-  const pets = useSelector((state: RootState) => state.petsToCompare); 
+
+  const pets: PetsToCompareList[] = useSelector((state: RootState) => state.comparison.petsToCompare);
 
   useEffect(() => {
-    dispatch(findPetsById(ids.split('-'))); //run action, which fetch pets by id from database
-  });
+    dispatch(asyncGetPeysById.request(ids.split('-'))); //run action, which fetch pets by id from database
+  },[]);
+
   return (
     <div className="comparison-page">
       <Logo />
       <h1 className="header">Порівняння</h1>
-      <div className="characteristics">
-        <h2>Характеристики</h2>
-        <h3>Популярність</h3>
-        <Characteristics pets={ pets } display="popularity" rate={true}/>  {/* if we want to show Rate bar, rate = true */}
-        <h3>Тренування</h3>
-        <Characteristics pets={ pets } display="training" rate={true}/>
-      </div>
-      <div className="specificity-care">
-        <h2>Особливості догляду</h2>
-        <h3>Час піклування (хв. на день)</h3>
-        <Characteristics pets={ pets } display="careTime" rate={false}/> 
-      </div>
+      {pets.length > 0 && (
+        <>
+          <div className="characteristics">
+            <h2>Характеристики</h2>
+            <h3>Популярність</h3>
+            <Characteristics data={pets[0]} />
+            <h3>Тренування</h3>
+            <Characteristics data={pets[1]} />
+            <h3>Розмір</h3>
+            <Characteristics data={pets[2]} />
+            <h3>Вміння</h3>
+            <Characteristics data={pets[3]} />
+          </div>
+          <div className="specificity-care">
+            <h2>Особливості догляду</h2>
+            <h3>Час піклування (год. на день)</h3>
+            <Characteristics data={pets[14]} />
```

Comment:
I'd like to suggest creating separate variables for pets[0], pets[1] etc. It'll make your code more readable

Reply:
I think that `pets[14]` is alert for code smelling. Let's keep it as is, but should be improved in the nearest feature. Please consider to improve it with @okhabr suggestion, but also add comment to improve this approach in the feature.